Code/Modules/mutalisk/common.h

```diff
@@ -1,12 +1,14 @@
 #ifndef MUTALISK_COMMON_H_
 #define MUTALISK_COMMON_H_
 
+#include <mutant/io_factory.h>
+
 #include "cfg.h"
 #include "types.h"
 #include "metainfo.h"
 
 // macros
-#define MUTALISK_NOT_IMPLEMENTED(msg) ()
+#define MUTALISK_NOT_IMPLEMENTED(msg)
```

Comment:
In some instances (like platform selection) the use of this should be replaced by `#error`.

Reply:
feature changes should be approved by someone else.

Reply:
This change is pointless - the PSP codepath will never hit this define..